Web/Admin.UI/src/app/components/validation-config/validation-categories/edit-validation-category/edit-validation-category.component.ts

```
@@ -0,0 +1,218 @@
+    const dialogRef = this.dialog.open(RulesetAddEditDialogComponent, {
+      width: '830px',
+      panelClass: ['vd-dialog', 'ruleset-add-edit-dialog']
+    });
+    
+    // Set the properties directly on the component instance
+    dialogRef.componentInstance.dialogTitle = 'Edit Rule Set';
+    dialogRef.componentInstance.rules = rule ? [rule] : [];
+  }
+
+  onDelete(ruleSet?: IValidationRuleSet): void {
+    if (ruleSet && ruleSet.rules && ruleSet.rules.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)